Build 4.0.0 Docs

With the 4.0.0 release we need up to date docs.

Read The Docs has an error - Chasing up the next best step here: https://github.com/readthedocs/readthedocs.org/issues/6832

So `setup.py install` chooses to install the same module twice. I don't know why for sure, but here's a guess. 

So it's not actually bandersnatch's fault, even a test module fails to install with the following:
```
test
├── setup.cfg
├── setup.py
└── test.py
```

<details>
<summary>setup.cfg</summary>

```cfg
[metadata]
name = test
author = Richard Si

[options]
install_requires =
    aiohttp
    aiohttp-wsgi
    setuptools>40.0.0
```

</details>

<details>
<summary>setup.py</summary>

```py3
from setuptools import setup

setup()
```

</details>

<details>
<summary>test.py</summary>

```py3
print("hello, world")  # this is just a test module, what else do I put?
```

</details>

Note how I replaced `aiohttp-xmlrpc` with `aiohttp-wsgi`, a completely different dependency.

Now, let's try to install this useless module:
```console
(env) richard-26@ubuntu-laptop:~/programming/test/aiohttp_error$ python setup.py install
running install
running bdist_egg
running egg_info
creating test.egg-info
writing test.egg-info/PKG-INFO
writing dependency_links to test.egg-info/dependency_links.txt
writing requirements to test.egg-info/requires.txt
writing top-level names to test.egg-info/top_level.txt
writing manifest file 'test.egg-info/SOURCES.txt'
reading manifest file 'test.egg-info/SOURCES.txt'
writing manifest file 'test.egg-info/SOURCES.txt'
installing library code to build/bdist.linux-x86_64/egg
running install_lib
warning: install_lib: 'build/lib' does not exist -- no Python modules to install

creating build
creating build/bdist.linux-x86_64
creating build/bdist.linux-x86_64/egg
creating build/bdist.linux-x86_64/egg/EGG-INFO
copying test.egg-info/PKG-INFO -> build/bdist.linux-x86_64/egg/EGG-INFO
copying test.egg-info/SOURCES.txt -> build/bdist.linux-x86_64/egg/EGG-INFO
copying test.egg-info/dependency_links.txt -> build/bdist.linux-x86_64/egg/EGG-INFO
copying test.egg-info/requires.txt -> build/bdist.linux-x86_64/egg/EGG-INFO
copying test.egg-info/top_level.txt -> build/bdist.linux-x86_64/egg/EGG-INFO
zip_safe flag not set; analyzing archive contents...
creating dist
creating 'dist/test-0.0.0-py3.8.egg' and adding 'build/bdist.linux-x86_64/egg' to it
removing 'build/bdist.linux-x86_64/egg' (and everything under it)
Processing test-0.0.0-py3.8.egg
Copying test-0.0.0-py3.8.egg to /home/richard-26/programming/test/aiohttp_error/env/lib/python3.8/site-packages
Adding test 0.0.0 to easy-install.pth file

Installed /home/richard-26/programming/test/aiohttp_error/env/lib/python3.8/site-packages/test-0.0.0-py3.8.egg
Processing dependencies for test==0.0.0
Searching for aiohttp-wsgi
Reading https://pypi.org/simple/aiohttp-wsgi/
Downloading https://files.pythonhosted.org/packages/ad/30/210b391ec7fd3217bbcc9d95dad60aa56c7754c5d27526a64ef54adc8721/aiohttp_wsgi-0.8.2-py3-none-any.whl#sha256=25b143b4b64482b1e219fa1d3253916cf08a3fd1bd3f1aeb55cb2ff8b05e2e33
Best match: aiohttp-wsgi 0.8.2
Processing aiohttp_wsgi-0.8.2-py3-none-any.whl
Installing aiohttp_wsgi-0.8.2-py3-none-any.whl to /home/richard-26/programming/test/aiohttp_error/env/lib/python3.8/site-packages
writing requirements to /home/richard-26/programming/test/aiohttp_error/env/lib/python3.8/site-packages/aiohttp_wsgi-0.8.2-py3.8.egg/EGG-INFO/requires.txt
Adding aiohttp-wsgi 0.8.2 to easy-install.pth file
Installing aiohttp-wsgi-serve script to /home/richard-26/programming/test/aiohttp_error/env/bin

Installed /home/richard-26/programming/test/aiohttp_error/env/lib/python3.8/site-packages/aiohttp_wsgi-0.8.2-py3.8.egg
Searching for aiohttp
Downloading https://files.pythonhosted.org/packages/58/62/b8ff424672d7b708488068e947140d26e40b2d8c21a35121d8c9f405a8a0/aiohttp-wsgi-0.8.2.tar.gz#sha256=5d45a14a3f0fd54609ca7648cbf0362ff9077ef341a6ee9ff3f4ff9591d2289c
Best match: aiohttp wsgi-0.8.2
Processing aiohttp-wsgi-0.8.2.tar.gz
Writing /tmp/easy_install-hxwz15ju/aiohttp-wsgi-0.8.2/setup.cfg
Running aiohttp-wsgi-0.8.2/setup.py -q bdist_egg --dist-dir /tmp/easy_install-hxwz15ju/aiohttp-wsgi-0.8.2/egg-dist-tmp-8dopdjd9
zip_safe flag not set; analyzing archive contents...
removing '/home/richard-26/programming/test/aiohttp_error/env/lib/python3.8/site-packages/aiohttp_wsgi-0.8.2-py3.8.egg' (and everything under it)
Moving aiohttp_wsgi-0.8.2-py3.8.egg to /home/richard-26/programming/test/aiohttp_error/env/lib/python3.8/site-packages
aiohttp-wsgi 0.8.2 is already the active version in easy-install.pth
Installing aiohttp-wsgi-serve script to /home/richard-26/programming/test/aiohttp_error/env/bin

Installed /home/richard-26/programming/test/aiohttp_error/env/lib/python3.8/site-packages/aiohttp_wsgi-0.8.2-py3.8.egg
error: The 'aiohttp' distribution was not found and is required by test
```

And it still fails with the same error. First, it installs a `aiohttp-wgsi` wheel which reports as `aiohttp-wsgi 0.8.2`, then when it looks for `aiohttp` it installs a `aiohttp-wsgi` source which reports as `aiohttp wsgi-0.8.2`. I think it's the space causing the error. It thinks that it found `aiohttp`, but actually found the source of `aiohttp-wsgi`. I mean, aren't dashes a bad idea for module names? The `aiowsgi` wheel is named `aiohttp_wsgi-0.8.2-py3-none-any.whl ` while the source is named `aiohttp-wsgi-0.8.2.tar.gz `

A quick and dirty fix is to just flip the places of `aiohttp` and `aiohttp-wsgi`,

```cfg
[metadata]
name = test
author = Richard Si

[options]
install_requires =
    aiohttp-wsgi
    aiohttp
    setuptools>40.0.0
```

```console
(env) richard-26@ubuntu-laptop:~/programming/test/aiohttp_error$ python setup.py install 
running install
running bdist_egg
running egg_info
creating test.egg-info
writing test.egg-info/PKG-INFO
writing dependency_links to test.egg-info/dependency_links.txt
writing requirements to test.egg-info/requires.txt
writing top-level names to test.egg-info/top_level.txt
writing manifest file 'test.egg-info/SOURCES.txt'
reading manifest file 'test.egg-info/SOURCES.txt'
writing manifest file 'test.egg-info/SOURCES.txt'
installing library code to build/bdist.linux-x86_64/egg
running install_lib
warning: install_lib: 'build/lib' does not exist -- no Python modules to install

creating build
creating build/bdist.linux-x86_64
creating build/bdist.linux-x86_64/egg
creating build/bdist.linux-x86_64/egg/EGG-INFO
copying test.egg-info/PKG-INFO -> build/bdist.linux-x86_64/egg/EGG-INFO
copying test.egg-info/SOURCES.txt -> build/bdist.linux-x86_64/egg/EGG-INFO
copying test.egg-info/dependency_links.txt -> build/bdist.linux-x86_64/egg/EGG-INFO
copying test.egg-info/requires.txt -> build/bdist.linux-x86_64/egg/EGG-INFO
copying test.egg-info/top_level.txt -> build/bdist.linux-x86_64/egg/EGG-INFO
zip_safe flag not set; analyzing archive contents...
creating dist
creating 'dist/test-0.0.0-py3.8.egg' and adding 'build/bdist.linux-x86_64/egg' to it
removing 'build/bdist.linux-x86_64/egg' (and everything under it)
Processing test-0.0.0-py3.8.egg
Copying test-0.0.0-py3.8.egg to /home/richard-26/programming/test/aiohttp_error/env/lib/python3.8/site-packages
Adding test 0.0.0 to easy-install.pth file

Installed /home/richard-26/programming/test/aiohttp_error/env/lib/python3.8/site-packages/test-0.0.0-py3.8.egg
Processing dependencies for test==0.0.0
Searching for aiohttp
Reading https://pypi.org/simple/aiohttp/
Downloading https://files.pythonhosted.org/packages/f0/f2/703dba52c7620199ea0ec8ea9a4a2f06203b4893b94f60240c2c10225043/aiohttp-4.0.0a1.tar.gz#sha256=b5036133c1ba77ed5a70208d2a021a90b76fdf8bf523ae33dae46d4f4380d86f
Best match: aiohttp 4.0.0a1
Processing aiohttp-4.0.0a1.tar.gz
Writing /tmp/easy_install-1l1dh8z9/aiohttp-4.0.0a1/setup.cfg
Running aiohttp-4.0.0a1/setup.py -q bdist_egg --dist-dir /tmp/easy_install-1l1dh8z9/aiohttp-4.0.0a1/egg-dist-tmp-8ujtx_w9
**********************
* Accellerated build *
**********************
warning: no files found matching 'aiohttp' anywhere in distribution
warning: no previously-included files matching '*.pyc' found anywhere in distribution
warning: no previously-included files matching '*.pyd' found anywhere in distribution
warning: no previously-included files matching '*.so' found anywhere in distribution
warning: no previously-included files matching '*.lib' found anywhere in distribution
warning: no previously-included files matching '*.dll' found anywhere in distribution
warning: no previously-included files matching '*.a' found anywhere in distribution
warning: no previously-included files matching '*.obj' found anywhere in distribution
warning: no previously-included files found matching 'aiohttp/*.html'
no previously-included directories found matching 'docs/_build'
aiohttp/_http_parser.c: In function ‘__pyx_f_7aiohttp_12_http_parser_cb_on_header_field’:
aiohttp/_http_parser.c:11643:34: warning: comparison between signed and unsigned integer expressions [-Wsign-compare]
       __pyx_t_6 = ((__pyx_v_size > __pyx_v_pyparser->_max_field_size) != 0);
                                  ^
aiohttp/_http_parser.c:11739:148: warning: passing argument 2 of ‘__pyx_v_pyparser->__pyx_vtab->_on_header_field’ discards ‘const’ qualifier from pointer target type [-Wdiscarded-qualifiers]
 = ((struct __pyx_vtabstruct_7aiohttp_12_http_parser_HttpParser *)__pyx_v_pyparser->__pyx_vtab)->_on_header_field(__pyx_v_pyparser, __pyx_v_at, __pyx_v_length); if (unlikely(!__pyx_t_1)) __PYX_ERR(0, 638, __pyx_L3_error)
                                                                                                                                    ^~~~~~~~~~
aiohttp/_http_parser.c:11739:148: note: expected ‘char *’ but argument is of type ‘const char *’
aiohttp/_http_parser.c: In function ‘__pyx_f_7aiohttp_12_http_parser_cb_on_header_value’:
aiohttp/_http_parser.c:11961:34: warning: comparison between signed and unsigned integer expressions [-Wsign-compare]
       __pyx_t_6 = ((__pyx_v_size > __pyx_v_pyparser->_max_field_size) != 0);
                                  ^
aiohttp/_http_parser.c:12057:148: warning: passing argument 2 of ‘__pyx_v_pyparser->__pyx_vtab->_on_header_value’ discards ‘const’ qualifier from pointer target type [-Wdiscarded-qualifiers]
 = ((struct __pyx_vtabstruct_7aiohttp_12_http_parser_HttpParser *)__pyx_v_pyparser->__pyx_vtab)->_on_header_value(__pyx_v_pyparser, __pyx_v_at, __pyx_v_length); if (unlikely(!__pyx_t_1)) __PYX_ERR(0, 655, __pyx_L3_error)
                                                                                                                                    ^~~~~~~~~~
aiohttp/_http_parser.c:12057:148: note: expected ‘char *’ but argument is of type ‘const char *’
aiohttp/_find_header.c: In function ‘find_header’:
aiohttp/_find_header.c:21:19: warning: initialization discards ‘const’ qualifier from pointer target type [-Wdiscarded-qualifiers]
     char *pchar = str;
                   ^~~
aiohttp/_find_header.c:9997:1: warning: label ‘missing’ defined but not used [-Wunused-label]
 missing:
 ^~~~~~~
aiohttp/_find_header.c:27:1: warning: label ‘INITIAL’ defined but not used [-Wunused-label]
 INITIAL:
 ^~~~~~~
zip_safe flag not set; analyzing archive contents...
aiohttp.__pycache__._frozenlist.cpython-38: module references __file__
aiohttp.__pycache__._helpers.cpython-38: module references __file__
aiohttp.__pycache__._http_parser.cpython-38: module references __file__
aiohttp.__pycache__._http_writer.cpython-38: module references __file__
aiohttp.__pycache__._websocket.cpython-38: module references __file__
creating /home/richard-26/programming/test/aiohttp_error/env/lib/python3.8/site-packages/aiohttp-4.0.0a1-py3.8-linux-x86_64.egg
Extracting aiohttp-4.0.0a1-py3.8-linux-x86_64.egg to /home/richard-26/programming/test/aiohttp_error/env/lib/python3.8/site-packages
Adding aiohttp 4.0.0a1 to easy-install.pth file

Installed /home/richard-26/programming/test/aiohttp_error/env/lib/python3.8/site-packages/aiohttp-4.0.0a1-py3.8-linux-x86_64.egg
Searching for aiohttp-wsgi
Reading https://pypi.org/simple/aiohttp-wsgi/
Downloading https://files.pythonhosted.org/packages/ad/30/210b391ec7fd3217bbcc9d95dad60aa56c7754c5d27526a64ef54adc8721/aiohttp_wsgi-0.8.2-py3-none-any.whl#sha256=25b143b4b64482b1e219fa1d3253916cf08a3fd1bd3f1aeb55cb2ff8b05e2e33
Best match: aiohttp-wsgi 0.8.2
Processing aiohttp_wsgi-0.8.2-py3-none-any.whl
Installing aiohttp_wsgi-0.8.2-py3-none-any.whl to /home/richard-26/programming/test/aiohttp_error/env/lib/python3.8/site-packages
writing requirements to /home/richard-26/programming/test/aiohttp_error/env/lib/python3.8/site-packages/aiohttp_wsgi-0.8.2-py3.8.egg/EGG-INFO/requires.txt
Adding aiohttp-wsgi 0.8.2 to easy-install.pth file
Installing aiohttp-wsgi-serve script to /home/richard-26/programming/test/aiohttp_error/env/bin

Installed /home/richard-26/programming/test/aiohttp_error/env/lib/python3.8/site-packages/aiohttp_wsgi-0.8.2-py3.8.egg
Searching for async_timeout<4.0,>=3.0
Reading https://pypi.org/simple/async_timeout/
Downloading https://files.pythonhosted.org/packages/e1/1e/5a4441be21b0726c4464f3f23c8b19628372f606755a9d2e46c187e65ec4/async_timeout-3.0.1-py3-none-any.whl#sha256=4291ca197d287d274d0b6cb5d6f8f8f82d434ed288f962539ff18cc9012f9ea3
Best match: async-timeout 3.0.1
Processing async_timeout-3.0.1-py3-none-any.whl
Installing async_timeout-3.0.1-py3-none-any.whl to /home/richard-26/programming/test/aiohttp_error/env/lib/python3.8/site-packages
Adding async-timeout 3.0.1 to easy-install.pth file

Installed /home/richard-26/programming/test/aiohttp_error/env/lib/python3.8/site-packages/async_timeout-3.0.1-py3.8.egg
Searching for chardet<4.0,>=2.0
Reading https://pypi.org/simple/chardet/
Downloading https://files.pythonhosted.org/packages/bc/a9/01ffebfb562e4274b6487b4bb1ddec7ca55ec7510b22e4c51f14098443b8/chardet-3.0.4-py2.py3-none-any.whl#sha256=fc323ffcaeaed0e0a02bf4d117757b98aed530d9ed4531e3e15460124c106691
Best match: chardet 3.0.4
Processing chardet-3.0.4-py2.py3-none-any.whl
Installing chardet-3.0.4-py2.py3-none-any.whl to /home/richard-26/programming/test/aiohttp_error/env/lib/python3.8/site-packages
Adding chardet 3.0.4 to easy-install.pth file
Installing chardetect script to /home/richard-26/programming/test/aiohttp_error/env/bin

Installed /home/richard-26/programming/test/aiohttp_error/env/lib/python3.8/site-packages/chardet-3.0.4-py3.8.egg
Searching for attrs>=17.3.0
Reading https://pypi.org/simple/attrs/
Downloading https://files.pythonhosted.org/packages/a2/db/4313ab3be961f7a763066401fb77f7748373b6094076ae2bda2806988af6/attrs-19.3.0-py2.py3-none-any.whl#sha256=08a96c641c3a74e44eb59afb61a24f2cb9f4d7188748e76ba4bb5edfa3cb7d1c
Best match: attrs 19.3.0
Processing attrs-19.3.0-py2.py3-none-any.whl
Installing attrs-19.3.0-py2.py3-none-any.whl to /home/richard-26/programming/test/aiohttp_error/env/lib/python3.8/site-packages
writing requirements to /home/richard-26/programming/test/aiohttp_error/env/lib/python3.8/site-packages/attrs-19.3.0-py3.8.egg/EGG-INFO/requires.txt
Adding attrs 19.3.0 to easy-install.pth file

Installed /home/richard-26/programming/test/aiohttp_error/env/lib/python3.8/site-packages/attrs-19.3.0-py3.8.egg
error: aiohttp 4.0.0a1 is installed but aiohttp<4,>=3.4 is required by {'aiohttp-wsgi'}
```

Oh well, it looks like there's a dependency conflict, eh whatever, you get the point. This also works with `aiohttp-xmlrpc` I have no idea why this works; at least it works:

![Screenshot from 2020-06-05 15-52-08](https://user-images.githubusercontent.com/63936253/83917301-9ead6580-a744-11ea-9e43-829c8189de75.png)

I do have a PR ready by the way.

--- a/setup.cfg
+++ b/setup.cfg
@@ -20,8 +20,8 @@ version = 4.1.0.dev1
 
 [options]
 install_requires =
-    aiohttp
     aiohttp-xmlrpc
+    aiohttp
     filelock
     importlib_resources; python_version < '3.7'
     packaging

